frontend/src/components/AddTodo.tsx

```diff
@@ -13,14 +13,15 @@ export default function AddTodo(props: Props) {
         setDescription(event.target.value)
     }
 
-    function onSave() {
-        const newTodo: NewTodo = {description, status: "OPEN"}
+    function onSave(event: any) {
```

Comment:
-Use specific EventType.
- clearer name instead of onSave 
```suggestion
    function onSubmit(event: FormEvent<HTMLFormElement>) {
```